docs/htdocs/info/docs/tools/vep/script/vep_other.html

```diff
@@ -256,26 +256,27 @@ <h4 id="pick_options">Options</h4>
     <li style="margin-bottom:20px">
       <b>--pick</b></p>
       <p>This is the option
-      we anticipate will be of use. VEP chooses one block of
+      VEP chooses one block of
       annotation per variant, using an ordered set of criteria. This order may be
       customised using <a href="vep_options.html#opt_pick_order">--pick_order</a>.
       <ol>
+        <li><a href="http://www.ensembl.org/info/genome/genebuild/mane.html">MANE Select transcript status</a></li>
+        <li><a href="http://www.ensembl.org/info/genome/genebuild/mane.html">MANE Plus Clinical transcript status</a></li>
         <li>canonical status of transcript</li>
```

Comment:
Is this information only relevant to the www site? If so, please change the protocol to https - if not, make it a relative link.